Apps/Inc/Tasks.h

```diff
@@ -134,4 +134,7 @@ extern OS_SEM SendCarCAN_Sem4;
  */
 // TODO: Put all global state variables here
 
+extern float RequestedMotorRPM;
+extern float RequestedMotorCurrentPercent;
+
```

Comment:
I would restructure this to go with the new CarState struct

Reply:
The Car State struct currently holds velocity as a uint8_t, but the motor controller wants a float value. Moreover, motor controller drive command requires a value in RPM, not m/s, so I assume those fields in car state have to change

Reply:
Yeah so I would change the data types in the struct to match that. But we're just trying to get rid of these globals